### PR TITLE
Staging 0.34.1

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
> [!warning]
> **This is public repository, ensure not to disclose:**
> - [ ] personal data beyond what is necessary for interacting with this pull request
> - [ ] business confidential information, such as customer names

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [x] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [ ] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change  
- [ ] kind/dev-change    
- [ ] kind/security      
- [ ] [kind/adr]()       

### What does this PR do / why do we need this PR?

...

- Fixes #

#### Checklist

- [x] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packages in the `NetworkPolicy Dashboard`
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

File: helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh [hunk 1]
```diff
@@ -25,26 +28,23 @@ wait_for_db_ready() {
 
 dump_database() {
     echo "Dumping database"  >&2
-    pg_dump -U postgres -h "$PG_HOSTNAME" registry > ${backup_dir}/registry.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" postgres > ${backup_dir}/postgres.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" notarysigner > ${backup_dir}/notarysigner.back
-    pg_dump -U postgres -h "$PG_HOSTNAME" notaryserver > ${backup_dir}/notaryserver.back
+    pg_dump -U postgres -h "$PG_HOSTNAME" registry | gzip -c > "${dump_dir}/registry.back.gz"
+    pg_dump -U postgres -h "$PG_HOSTNAME" postgres | gzip -c > "${dump_dir}/postgres.back.gz"
```
Comment (Xartos) [inline helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh:32]: We need to add this commit as well https://github.com/elastisys/compliantkubernetes-apps/commit/3abe5f393ebfca77bc3a4a1ae85581fd8b5c5a2f to make it work for apps < 0.35

**Review by Xartos — CHANGES_REQUESTED** [review]

File: helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh (hunk 1 above)
Comment (Pavan-Gunda) [reply helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh:32]: Done !!

**Review by Xartos — APPROVED** [review]

File: changelog/0.34.md [hunk 2]
```diff
@@ -99,3 +99,18 @@ Released 2023-11-21
...
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+> - Fixes issue where log-manager compaction script isn't removing all files as it should
+
+## Changes by kind
+
+### Other(s)
+
+- [#1863](https://github.com/elastisys/compliantkubernetes-apps/pull/1863) - bug: apps-sc: Fixed storage leak in log-manager compaction @Xartos
```
Comment (davidumea) [inline changelog/0.34.md:115]:
```suggestion
- [#1863](https://github.com/elastisys/compliantkubernetes-apps/pull/1863) - bug: Fixed storage leak in log-manager compaction @Xartos
```

File: changelog/0.34.md [hunk 3]
```diff
@@ -99,3 +99,18 @@ Released 2023-11-21
...
+
+Released 2023-12-19
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+> - Fixes issue where log-manager compaction script isn't removing all files as it should
+
+## Changes by kind
+
+### Other(s)
```
Comment (davidumea) [inline changelog/0.34.md:113]: Also shouldn't these be under kind "Bug(s)" instead of "Other(s)"?

Comment (Pavan-Gunda) [reply changelog/0.34.md:113]: Yeah !! I can move these to bugs

Comment (Pavan-Gunda) [reply changelog/0.34.md:113]: I see that everything other than features and improvements are put under others by the release script.

**Review by davidumea — APPROVED** [review]

**Review by lucianvlad — APPROVED** [review]

**Comment by crssnd** [comment]
maybe we should also fix the trivy-operator chart version in this patch?

**Review by anders-elastisys — APPROVED** [review]

**Review by viktor-f — APPROVED** [review]

**Pull request merged**